mllib/src/test/scala/org/apache/spark/ml/tuning/CrossValidatorSuite.scala

```diff
@@ -120,6 +120,33 @@ class CrossValidatorSuite
     }
   }
 
+  test("cross validation with parallel evaluation") {
+    val lr = new LogisticRegression
+    val lrParamMaps = new ParamGridBuilder()
+      .addGrid(lr.regParam, Array(0.001, 1000.0))
+      .addGrid(lr.maxIter, Array(0, 3))
+      .build()
+    val eval = new BinaryClassificationEvaluator
+    val cv = new CrossValidator()
+      .setEstimator(lr)
+      .setEstimatorParamMaps(lrParamMaps)
+      .setEvaluator(eval)
+      .setNumFolds(2)
+      .setParallelism(1)
+    val cvSerialModel = cv.fit(dataset)
+    cv.setParallelism(2)
```

Comment:
how do we validate setParallelism is parallelizing?

Reply:
It's a little difficult to do this in a unit test without making it flaky.  I have run tests manually and verified it is working by both the expected speedup in timing and that the expected number of tasks are run concurrently.  I can post some results if that would help.

Reply:
@BryanCutler may be worth posting the result to the JIRA for posterity.